snapshot/policy/error_handling_policy.go

```diff
@@ -0,0 +1,33 @@
+package policy
+
+// ErrorHandlingPolicy describes files to be ignored when taking snapshots.
+type ErrorHandlingPolicy struct {
+	IgnoreFileErrors         bool `json:"ignoreFileErrs,omitempty"`
+	IgnoreFileErrorsSet      bool `json:"ignoreFileErrsSet,omitempty"`
+	NoParentIgnoreFileErrors bool `json:"noParentIgnoreFileErrs,omitempty"`
+
+	IgnoreDirectoryErrors         bool `json:"ignoreDirErrs,omitempty"`
+	IgnoreDirectoryErrorsSet      bool `json:"ignoreDirErrsSet,omitempty"`
+	NoParentIgnoreDirectoryErrors bool `json:"noParentIgnoreDirErrs,omitempty"`
+}
+
+// Merge applies default values from the provided policy.
```

Comment:
I suppose you could interpret them as default values, as in "If the value isn't explicitly set in this policy, it will default to some other policy's value". This is the same wording used for the `Merge` method for all other policy sections (`FilesPolicy`, `CompressionPolicy`, `RetentionPolicy`, `SchedulingPolicy`)